src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main
+      class="card"
+      data-qa="card"
+    >
+      <img src="images/imac.jpeg"
+        alt="Apple A1419"
+        class="img card__img"
+      >
+      <section class="card__article-information">
+        <h1 class="card__name">
```

Comment:
Use a less specific heading tag, like 'h2', for the product name. Reserve 'h1' for the main title on the page.